ci/min_deps.yaml

```diff
@@ -0,0 +1,18 @@
+name: min_deps
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - astropy>=4.0
+  - coverage
+  - h5py
+  - numpy>=1.15
```

Comment:
If we're testing min deps, shouldnt we be enforcing installation of the minimum deps, rather than saying anything above a certain min?

Reply:
we could. This is how we do it other places, mainly I was wanting to enforce that tests are skipped properly so that you can run tests if you don't install any extras.

Reply:
So it was minimal packages rather than minimal versions I was targeting. But we could do the latter as well.

Reply:
Ahhhh, I see. I do think an actual minimum version dep test might be a good idea (perhaps not on every PR?) but we can do that another time.